kratos/python_scripts/multiple_points_output_process.py

```diff
@@ -0,0 +1,91 @@
+from __future__ import print_function, absolute_import, division  # makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+# Importing the Kratos Library
+import KratosMultiphysics
+
+# other imports
+from point_output_process import PointOutputProcess
+import os
+
+def Factory(settings, Model):
+    if(type(settings) != KratosMultiphysics.Parameters):
+        raise Exception("expected input shall be a Parameters object, encapsulating a json string")
+    return MultiplePointsOutputProcess(Model, settings["Parameters"])
+
+class MultiplePointsOutputProcess(KratosMultiphysics.Process):
+    """This process writes several points to a file
+    Internally it holds objects of type "PointOutputProcess"
+    Usage:
+        - directly by specifying the locations of the points for which output is wanted
+        - inside other processes that create points for which output is required
+          (e.g. LinePointsOutputProcess)
+    """
+    def __init__(self, model, params):
+
+        default_settings = KratosMultiphysics.Parameters('''{
+            "model_part_name"   : "",
+            "entity_type"       : "element",
+            "positions"         : [[]],
+            "output_variables"  : [],
+            "output_file_name"  : "",
+            "write_buffer_size" : -1,
+            "print_format"      : ""
+        }''')
+
+        params.ValidateAndAssignDefaults(default_settings)
+
+        positions = params["positions"].GetMatrix()
+
+        num_points = positions.Size1()
+        if num_points == 0:
+            raise Exception('No positions were specified"')
+        if positions.Size2() != 3:
+            raise Exception('The positions have to be provided with 3 coordinates!')
+
+        self.point_output_processes = []
+        params.RemoveValue("positions")
+        params.AddEmptyValue("position")
+        position_vec = KratosMultiphysics.Vector(3)
+      
+        if params["output_file_name"].GetString().endswith(".dat"):
```

Comment:
is this needed?
I think the `PointOutputProcess` takes care of that

Reply:
In the multiple_points_output_process.py the check in line 50 is needed, as it adds an index, which would be bad if it were to be added before an exntesion ".dat".

Reply:
ok